scripts/python/check_jbod_psu_status.py

```diff
@@ -0,0 +1,87 @@
+from integralstor import command, alerts
+import re, sys
+
+def get_enclosure_devices():
+    # Returns a list of /dev/sgX path for Connected Storage Enclosures
+    enclosures = []
+    try:
+        sg_devs, err = command.get_command_output('sg_map -i')
+        if err:
+            raise Exception(err)
+
+        if sg_devs:
+            for sg_dev in sg_devs:
+                    encl = re.findall('ENCL', sg_dev, re.IGNORECASE)
+                    if encl:
+                        res = re.findall(r'^\/\w+\/\w+', sg_dev)
+                        enclosures += res
```

Comment:
We need to check if res has a value before appending it to enclosures..

Reply:
Okay, I'll add a check before appending it.